api/src/main/java/io/strimzi/api/kafka/model/common/ClientTls.java

```diff
@@ -56,4 +58,11 @@ public void setAdditionalProperty(String name, Object value) {
         }
         this.additionalProperties.put(name, value);
     }
+
+    public String toVarString() {
+        Objects.requireNonNull(this.trustedCertificates, "trustedCertificates cannot be null");
+        return this.trustedCertificates.stream()
+                .map(certSecretSource -> certSecretSource.getSecretName() + "/" + certSecretSource.getCertificate())
+                .collect(Collectors.joining(";"));
+    }
```

Comment:
I don't really follow why this is added here. It does not seem to belong into the `api` I think.

Reply:
It was to generalise the code and reuse across `Bridge` and `Connector` since a lot of code was duplicated, so I refactored it and wanted to put it elsewhere than in one of those two, any suggestion?

Reply:
Moved it to cluster-operator to `ModelUtils.java`